lib/phobos_db_checkpoint.rb

```diff
@@ -20,48 +20,69 @@ module PhobosDBCheckpoint
 
   class << self
     attr_reader :db_config
-    attr_accessor :db_config_path, :db_dir, :migration_path
+    attr_accessor :db_config_path, :db_dir, :migration_path, :verbose
 
     # :nodoc:
     # ActiveRecord hook
     def table_name_prefix
       :phobos_db_checkpoint_
     end
 
-    def configure(pool_size: nil)
-      load_db_config(pool_size: pool_size)
+    def configure(options = {})
+      warn Kernel.caller.first + "[DEPRECATION] options are deprecated, use configuration files instead" if options.keys.any?
+      @verbose && puts("PhobosDBCheckpoint#configure")
+      load_db_config
       at_exit { PhobosDBCheckpoint.close_db_connection }
-      ActiveRecord::Base.establish_connection(db_config)
+      PhobosDBCheckpoint.establish_db_connection
     end
 
     def env
       ENV['RAILS_ENV'] ||= ENV['RACK_ENV'] ||= 'development'
     end
 
-    def load_db_config(pool_size: nil)
+    def load_db_config(options = {})
+      warn Kernel.caller.first + "[DEPRECATION] options are deprecated, use configuration files instead" if options.keys.any?
```

Comment:
Same as in previous case - missing space between the method and the rest of the text